x-pack/plugins/uptime/public/components/monitor/ping_list/columns/ping_timestamp.tsx

```diff
@@ -140,17 +150,36 @@ export const PingTimestamp = ({ timestamp, ping }: Props) => {
   );
 
   return (
-    <StepDiv ref={intersectionRef}>
+    <StepDiv
+      onMouseEnter={() => setIsImagePopoverOpen(true)}
```

Comment:
This solution results in the popover displaying the larger image when the mouse enters the `figcaption` element that the image contains as well. I wasn't sure if this functionality would be an issue. I did explore a few avenues  around this, but they weren't as simple as this. I also considered the notion that the caption should be semantically linked to the `img` itself as further justification.

I've included a GIF in the description illustrating the caption hover triggering the display.